sig-service-catalog/charter.md

```diff
@@ -0,0 +1,159 @@
+# SIG Service Catalog Charter
+Service Catalog is a Kubernetes extension project that implements the [Open
+Service Broker API](https://www.openservicebrokerapi.org/) (OSBAPI). It allows
+application developers the ability to provision and consume cloud services
+natively from within Kubernetes.
+
+This charter adheres to the conventions described in the [Kubernetes Charter
+README](https://github.com/kubernetes/community/blob/master/committee-steering/governance/README.md).
+
+## Scope
+
+### In scope
+
+This SIG’s main goals are:
+- Support, and adhere to, the Platform requirements of the [OSBAPI
+  specification](https://github.com/openservicebrokerapi/servicebroker/blob/master/spec.md).
+- Provide a UX for Kubernetes users that is consistent with both the OSB API
+  specification and traditional Kubernetes user interactions.
+- Align with the OSBAPI specification as changes are made.
+- Provide feedback (bugs or feature requests) to the OSBAPI WG.
+
+### Out of scope
+
+The following, non-exhaustive, items are out of scope:
+- Operation of OSBAPI Service Brokers.
+
+## Roles
+
+- Maintainers
+  - Maintainer is equivalent to the standard Kubernetes definition of Approver.
+  - Responsible for approving, and reviewing, pull requests.
+  - Responsible for technical planning and stewardship of the project.
+  - New maintainers may be nominated by a chair, and accepted via lazy
+    two-thirds resolution amongst the chairs.
+  - Maintainers may be nominated for removal from their position by a chair,
+    and accepted via lazy two-thirds resolution amongst the chairs.
+  - Maintainers may propose changes to this charter at any time, by submitting a
+    pull request and then notifying the SIG mailing list, to be accepted via
+    lazy two-thirds resolution amongst the chairs.
+
+- Chairs
+  - All maintainer’s roles.
+  - Chairs are listed in our [SIG
+    definition](https://github.com/kubernetes/community/tree/master/sig-service-catalog#chairs).
+  - Responsible for project administration activities within the SIG and are
+    non-technical in nature, such as organizing the weekly meetings.
+  - A chair does not have more rights, or votes, than a maintainer.
+  - Responsible for reporting the SIG’s status to the appropriate Kubernetes
+    leadership teams.
+  - All decisions amongst chairs are made using lazy consensus with a fallback
+    to a 2/3 majority vote (lazy two-thirds resolution).
+    This process is used for all decisions, such as changing chairs/maintainers
+    or modifying this charter.
+  - Chairs may nominate a new chair at any time, to be accepted via lazy
+    two-thirds resolution amongst the chairs.
+  - Chairs may decide to step down at any time.
+  - Chairs must remain active in the role and may be removed from the position
```

Comment:
Clarify if there is any other process for a chair to be removed other than by other chairs discussing.  I.e. can the maintainers decide chairs are out?

Reply:
@smarterclayton Do you have an example from another charter or suggestion about how maintainers could vote out an inactive chair? Currently we don't have a process for that though I could easily see it happening where multiple chairs are inactive and there's no defined way to fix that.

Reply:
I think we should have this, but the current charter was "how things are" not "how we wish they would be".